linera-views/README.md

```diff
@@ -7,6 +7,50 @@ atomically persisted in storage).
 
 <!-- cargo-rdme end -->
 
+The package provides essentially two functionalities:
+* An abstraction to access databases.
+* Several containers named views for storing data modeled on classical ones.
+
+See `DESIGN.md` for more details.
+
+## The supported databases.
```

Comment:
Remove the dots at the end of titles?

Can we move these sections to the inline doc in `lib.rs`? They will be brought back above by cargo-rdme

Reply:
Ok, done.